main.py

```diff
@@ -26,36 +29,265 @@ def info() -> Dict[str, List[str]]:
     the list of available image files."""
     list_of_images = list_images()
     list_of_models = Configuration.models
-    data = {"models": list_of_models, "images": list_of_images}
+    data = {
+        "models": list_of_models,
+        "images": list_of_images
+    }
     return data
 
 
 @app.get("/", response_class=HTMLResponse)
 def home(request: Request):
     """The home page of the service."""
-    return templates.TemplateResponse("home.html", {"request": request})
+    return templates.TemplateResponse(
+        "home.html",
+        {
+            "request": request
+        }
+    )
 
 
 @app.get("/classifications")
 def create_classify(request: Request):
     return templates.TemplateResponse(
         "classification_select.html",
-        {"request": request, "images": list_images(), "models": Configuration.models},
+        {
+            "request": request,
+            "images": list_images(),
+            "models": Configuration.models,
+            "userImage": 0
+        },
     )
 
 
 @app.post("/classifications")
-async def request_classification(request: Request):
+async def request_classification(
+    request: Request
+):
     form = ClassificationForm(request)
     await form.load_data()
     image_id = form.image_id
     model_id = form.model_id
     classification_scores = classify_image(model_id=model_id, img_id=image_id)
+    with open('classification_scores.json', 'w') as f:
+        json.dump(classification_scores, f)
```

Comment:
What happens if multiple users ask for a classification job? The file would be overwritten. Isn't it best to return the results directly? Or, if you reuse them later for other things, another way to do it would be to store the results with a unique name and clean up after the response is returned to the user (or after a while anyway).